### PR TITLE
feat: add cookie that tracks last views post ids

### PR DESCRIPTION
**Pull request by omnisite**

### CODE REVIEW

File: src/Bootstrap.php [hunk 1]
```diff
@@ -7,6 +7,15 @@
 
 namespace AchttienVijftien\Plugin\LastViewed;
 
+require plugin_dir_path( __FILE__ ) . 'Tracker.php';
```
Comment (dennisenderink) [inline src/Bootstrap.php:10]: These aren't needed.

File: src/View.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	 * Name of tracking cookie.
+	 */
+	public const TRACKING_COOKIE = '1815_last-viewed';
+
+	/**
+	 * Output posts found in the tracking cookie
+	 */
+	public function output_posts(): string {
+		// bail early if no or empty cookie
+		if ( empty( $_COOKIE[ self::TRACKING_COOKIE ] ) ) {
```
Comment (dennisenderink) [inline src/View.php]: Use constant from `Tracker`, like so: `Tracker::TRACKING_COOKIE`.

File: src/Shortcode.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * Tracks page views on single posts.
+ *
+ * @package AchttienVijftien\Plugin\LastViewed
+ */
+class Shortcode {
+
+	/**
+	 * Name of tracking cookie.
+	 */
+	public const TRACKING_COOKIE = '1815_last-viewed';
```
Comment (dennisenderink) [inline src/Shortcode.php]: Unused.

File: src/Shortcode.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ *
+ * @package AchttienVijftien\Plugin\LastViewed
+ */
+
+namespace AchttienVijftien\Plugin\LastViewed;
+
+use AchttienVijftien\Plugin\LastViewed\View;
+
+/**
+ * Tracks page views on single posts.
```
Comment (dennisenderink) [inline src/Shortcode.php]: Update description.

File: src/Bootstrap.php (hunk 1 above)
Comment (omnisite) [reply src/Bootstrap.php:10]: I thought so as well, but after activating the plugin, I got a fatal about not finding the classes.

Comment (dennisenderink) [reply src/Bootstrap.php:10]: Double checked, the classes are added to the psr4 autoload file.

Comment (omnisite) [reply src/Bootstrap.php:10]: I need this to work as if someone places these files in their plugin folder. That is what I did, and then I get fatals after activating. What file can we add to make everything autoload without someone having to do anything, but place the files?

File: src/View.php (hunk 2 above)
Comment (omnisite) [reply src/View.php]: done

File: src/Shortcode.php (hunk 3 above)
Comment (omnisite) [reply src/Shortcode.php]: done

File: src/Shortcode.php (hunk 4 above)
Comment (omnisite) [reply src/Shortcode.php]: done

**Review by dennisenderink — APPROVED** [review]

**Pull request merged**